.github/workflows/e2e-ios.yml

```diff
@@ -29,6 +29,10 @@ jobs:
           secrets: |-
             EMERGE_API_TOKEN:projects/1027349420744/secrets/EMERGE_API_TOKEN
       - uses: actions/checkout@v3
+        with:
+          # checkout PR head instead of merge commit so we can push changes on renovate PRs
+          # https://github.com/actions/checkout/tree/v3.6.0#checkout-pull-request-head-commit-instead-of-merge-commit
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
Should we make this conditional base on the renovatebot user then?

Reply:
I ended up doing the latter, although it got a bit messy. lmk what you think